src/components/tables/RowActions/AccessGrants/GrantWhatIsChanging.tsx

```diff
@@ -0,0 +1,42 @@
+import { Stack } from '@mui/material';
+import { useIntl } from 'react-intl';
+import { GrantWhatIsChangingProps } from './types';
+
+function GrantWhatIsChanging({
```

Comment:
The left side column renderer. I felt like highlighting the fields in bold was important... so annoying to have an entire little component just for that. But oh well.